Android.mk

```diff
@@ -59,6 +59,8 @@ LOCAL_CFLAGS += -DUSER_SPACE
 LOCAL_CFLAGS += -DTA_DIR=\"/system/lib/optee_armtz\"
 LOCAL_CFLAGS += -pthread
 
+LOCAL_ADDITIONAL_DEPENDENCIES := $(TA_DEV_KIT_DIR)/host_include/tee_api_types.h
```

Comment:
@d3zd3z @kuscsik would this work for you also? I'm thinking about the email discussion we had just a week or two ago regarding the `TA_DEV_KIT_DIR` make file argument?